win32/winapp.cpp

```diff
@@ -487,6 +487,7 @@ tWin32OS oeWin32Application::version(int *major, int *minor, int *build, char *s
 
   osinfo.dwOSVersionInfoSize = sizeof(osinfo);
 
+#pragma warning(suppress: 28159 4996)
```

Comment:
This is pretty opaque, please add a comment saying what this warning is, and why we disable it

Reply:
Guess it's about deprecation warning about GetVersion() call. I think this can be greatly simplified as we assume that we running on Windows 10 or later.

Reply:
yep, exactly. added a comment in https://github.com/DescentDevelopers/Descent3/pull/428/commits/ef33834a2a7c1b0f26a446de550d856bfb8c9825 to address the opacity of it tho

Reply:
thanks for the comment